<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>208</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>208</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>208</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Minimizer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEditProcess">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Process name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QKeySequenceEdit" name="hotkeyMinimize">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QKeySequenceEdit" name="hotkeyMaximize">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="apply">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxLaunchAtStartup">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Launch to tray with Windows</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetProcesses">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddProcess">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemoveProcess">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelectProcess">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
